src/app/CheckError.js

```diff
@@ -10,6 +13,77 @@ const CheckError = {
     MISSING_OBJECT: 8,
     DUPLICATED: 9,
     LOGIN_2FA_REQUIRED: 10,
+    CONFLICT: 11,
+    PUBLISHED_REPORT: 12,
+  },
+  messages: {
+    UNAUTHORIZED: (<FormattedMessage
```

Comment:
@caiosba Can I get a review of all the messages in this object to make sure they make sense? I was guessing for some of these because they aren't actually used in `check-api` at the moment.

Reply:
@dariusk yes, we're not using all of them currently, but the messages make sense!